openmoc/plotter.py

```diff
@@ -540,10 +581,11 @@ def plot_spatial_fluxes(solver, energy_groups=[1], norm=False,
     plot_params.geometry = geometry
     plot_params.zcoord = zcoord
     plot_params.gridsize = gridsize
+    plot_params.library = library
     plot_params.xlim = xlim
     plot_params.ylim = ylim
     plot_params.colorbar = True
-    plot_params.cmap = None
+    plot_params.cmap = plt.get_cmap('jet')
```

Comment:
We use matplotlib's default colormap for all plots except the flux and fission rates (or if one specifies one to `PlotParams` for `plot_spatial_data(...)`). Because of this I decided to keep the default as `None` and only specify `jet` for the two plot types where we want it.